src/gui/src/mainWindow.cpp

```diff
@@ -112,7 +112,8 @@ MainWindow::MainWindow(bool load_settings, QWidget* parent)
       charts_widget_(new ChartsWidget(this)),
       help_widget_(new HelpWidget(this)),
       find_dialog_(new FindObjectDialog(this)),
-      goto_dialog_(new GotoLocationDialog(this, viewers_))
+      goto_dialog_(new GotoLocationDialog(this, viewers_)),
+      window_title_(Gui::get()->getMainWindowTitle())
```

Comment:
this feels clunky, couldn't you just pass this value instead?

Reply:
I agree it's clunky. Rather than adding a new parameter to the constructor we can get the initial value through `setTitle` like we get subsequent updates